### DISCUSSION TITLE
which file is 'cluster log'?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#12116] [Q&A]
Run cmd 'ceph health detail' print:
"xx large omap objects " and "Search the cluster log for 'Large omap object found' for more details"

I want to  know which file is "cluster log" file?

**Comment by travisn** [comment]
This message is from ceph. See the Ceph docs for the description of the [cluster log](https://docs.ceph.com/en/latest/dev/logging/#use-of-the-cluster-log). It should just be the pod logs in Rook's case.

**Reply by microyahoo** [reply]
`large omap objects` output is in `/var/lib/rook/<namespace>/log/ceph-osd.xx.log` like below
![20230422-183337](https://user-images.githubusercontent.com/10202906/233778896-25fc1547-5a98-4df7-8a8f-c21c8f142742.png)

**Reply by galexrt** [reply]
@microyahoo It is the logs of the pod itself not the files on disk. Use `kubectl logs -n rook-ceph POD_NAME` to get the component's logs.

**Reply by microyahoo** [reply]
yes, we persist the log to disk.
